tokio/Cargo.toml

```diff
@@ -91,7 +91,7 @@ tokio-macros = { version = "~2.5.0", path = "../tokio-macros", optional = true }
 pin-project-lite = "0.2.11"
 
 # Everything else is optional...
-bytes = { version = "1.1.0", optional = true }
+bytes = { version = "1.10.0", optional = true }
```

Comment:
What is this bump for?

Reply:
To have access to `as_uninit_slice_mut`. Although I think I can bump to `1.4.0` if bumping to latest is undesirable. Without this, the `minimal-versions` job will fail.

Reply:
Yes, please bump to the minimum you can.

Reply:
Bumped it to `1.2.1`.